tts/Object.def.lua

```diff
@@ -506,6 +508,9 @@ function Object.isSmoothMoving() end
 ---@return boolean
 function Object.jointTo(object, parameters) end
 
+---@param object tts__Object
```

Comment:
Could you please add:
```lua
---@return boolean
```
It'll return `false` if someone is currently searching the deck or if the container or object being inserted are frozen. Probably some other cases too.

Reply:
Done.